SitecorePackageDeployer/Tasks/InstallPackage.cs

```diff
@@ -83,22 +84,23 @@ internal static string GetPackageSource()
             return packageSource;
         }
 
-        public void Run()
+        public List<SitecorePackage> Run()
         {
-            InstallPackages();
+            return InstallPackages();
         }
 
         /// <summary>
         /// Installs the packages found in the package source folder.
         /// </summary>
-        private void InstallPackages()
+        private List<SitecorePackage> InstallPackages()
         {
+            var packages = new List<SitecorePackage>();
             //Check to see if there is a post-step pending, and skip package install if there is
             if (File.Exists(Path.Combine(_packageSource, STARTUP_POST_STEP_PACKAGE_FILENAME)))
```

Comment:
Don't need to address in your PR, but I'm thinking of leaving this chunk in place, and when "force=1" is passed in, it should not only reset the install flag, but also delete this file if it exists- so it's a TRUE force.